scripts/trim_sdk.py

```diff
@@ -155,7 +160,7 @@ def main():
     _print_folder_size(mgmt_sdk_dir)
 
     # Removed unused API versions
-    resource_types = _get_biggest_sdks_to_trim()
+    resource_types = _get_all_sdks_to_trim()
```

Comment:
The list for top 10 biggest SDKs (L123~L148) can be removed, since we are not using them anymore.